public/app/plugins/datasource/graphite/datasource_integration.test.ts

```diff
@@ -0,0 +1,154 @@
+import { GraphiteDatasource } from './datasource';
+import { TemplateSrv } from 'app/features/templating/template_srv';
+import { BackendSrv } from 'app/core/services/backend_srv';
+import { of } from 'rxjs';
+import { ContextSrv, User } from '../../../core/services/context_srv';
+import { setBackendSrv } from '@grafana/runtime';
+
+jest.mock('@grafana/runtime', () => ({
+  ...(jest.requireActual('@grafana/runtime') as unknown as object),
```

Comment:
Wonder why is this here? If I understand this does sort of nothing as it mocks runtime with itself?

Reply:
I see, yes, we don't need this. I saw that it had been used in /app/plugins/datasource/graphite/datasource.test.ts but I think this was to replace the backendSrv with the imported version in @grafana/runtime. In the integration test, it doesn't do anything so I'll take it out.